test/create_job.test.js

```diff
@@ -0,0 +1,75 @@
+import { assert } from 'chai';
+import { run } from 'syncano-test';
+import dotenv from 'dotenv';
+
+import ElasticTranscoder from '../src/utils/ElasticTranscoder';
+
+dotenv.config();
+
+describe('create_job', () => {
+  const config = {
+    AWS_ACCESS_KEY_ID: process.env.AWS_ACCESS_KEY_ID,
+    AWS_SECRET_ACCESS_KEY: process.env.AWS_SECRET_ACCESS_KEY,
+    AWS_REGION: process.env.AWS_REGION,
```

Comment:
Destructure the envs with `{} = process.env` (applies to all the tests). Also it looks like the config is used in each test so it could be abstracted away.